api/tests/src/legacy_soap_api/test_legacy_soap_api_client.py

```diff
@@ -0,0 +1,11 @@
+from unittest import TestCase
+
+from src.legacy_soap_api.legacy_soap_api_client import LegacySOAPClient
+
+
+class TestLegacySoapAPIClient(TestCase):
+    def setUp(cls) -> None:
+        cls.client = LegacySOAPClient()
```

Comment:
I'd caution against using unittest, we use pytest for everything, and I don't know what behavior we would expect by mixing them.

Instead of setup, you can do something like:
```
class MyTestClass:
     @pytest.fixture(scope="class")
     def legacy_soap_client(self):
           return LegacySOAPClient()

     def test_thing(self, legacy_soap_client):
           ...
```